project/profiles/capacity4more/behat/features/contact.feature

```diff
@@ -16,3 +16,26 @@ Feature: Contact
     And I fill in "E-mail addresses" with "info@example.com, other@example.com"
     And I press "Save configuration"
     Then I should see "The configuration options have been saved."
+
+  @api
+  Scenario: Check contact page as anonymous
+    Given I am an anonymous user
+    When I go to "contact"
+    Then I should see an enabled "#edit-name" element
```

Comment:
Please change the tests in a way of a real scenario when user try to submit the form